pyincore/analyses/transportationrecovery/transportationrecoveryutil.py

```diff
@@ -152,7 +152,7 @@ def traveltime_freeflow(temp_network):
             for j in range(num_node):
                 if i != j:
                     if distance[i][j] == 0:
-                        distance[i][j] = float("inf")
+                        distance[i][j] = math.inf
                     travel_efficiency += 1 / distance[i][j]
```

Comment:
we might want to fix the test for this analysis. some id is missing
Could you look at the jupyter notebook and find out the correct id?